### PR TITLE
First jab at emulating std::experimental::simd types

### PR DESCRIPTION
**Pull request by jfalcou**

### CODE REVIEW

File: include/eve/std.hpp [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  template<typename T>
+  using simd_mask = eve::logical<eve::wide<T, eve::fixed<16/sizeof(T)>>>;
+
+  template<typename T, int N >
+  using fixed_size_simd_mask = eve::logical<eve::wide<T, eve::fixed<N>>>;
+
+  template<typename T>
+  using native_simd_mask = eve::experimental::simd_mask<T>;
+}
```
Comment (DenisYaroshevskiy) [inline include/eve/std.hpp:32]: New line

File: include/eve/std.hpp [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+//==================================================================================================
+#pragma once
+
+#include <eve/wide.hpp>
+#include <eve/logical.hpp>
+
+namespace eve::experimental
+{
+  template<typename T>
+  using simd = eve::wide<T, eve::fixed<16/sizeof(T)>>;
```
Comment (DenisYaroshevskiy) [inline include/eve/std.hpp:16]: Rly? Always 16?

**Review by DenisYaroshevskiy — COMMENTED** [review]
Seems reasonable

File: include/eve/std.hpp (hunk 2 above)
Comment (jfalcou) [reply include/eve/std.hpp:16]: I dunno, clang and g++ gives different size. I purposedly let this and I await the failure to update accordingly.

File: include/eve/std.hpp (hunk 1 above)
Comment (jfalcou) [reply include/eve/std.hpp:32]: OK

File: include/eve/std.hpp (hunk 2 above)
Comment (DenisYaroshevskiy) [reply include/eve/std.hpp:16]: OK. Let's do that.

**Comment by jfalcou** [comment]
Looks like it passes.
@DenisYaroshevskiy your opinion ?

**Review by DenisYaroshevskiy — APPROVED** [review]

**Pull request merged**